related_resources/contexts/BCMinesActPermit/v1.jsonld

```diff
@@ -1,6 +1,13 @@
 {
     "@context": {
         "@protected": true,
-        "BCMinesActPermitCredential": "https://bcgov.github.io/digital-trust-toolkit/docs/governance/mining/bc-mines-act-permit/1.1.1/governance"
+        "BCMinesActPermitCredential": "https://bcgov.github.io/digital-trust-toolkit/docs/governance/mining/bc-mines-act-permit/1.1.1/governance",
+        "BCMinesActPermit": {
+            "@id": "https://bcgov.github.io/digital-trust-toolkit/docs/governance/mining/bc-mines-act-permit/1.1.1",
```

Comment:
```suggestion
            "@id": "https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01#section10",
```

Reply:
this credential is not an actual permit, just an assurance and it has limitations. I think linking to the law is misleading.

Reply:
Good point, this needs to be defined in the governance document somewhere

Reply:
It is, at the file I've linked to. 

unless we need a separate file to separate the anoncred meaning and the untp dcc meaning.

Reply:
You cannot have 2 attributes with the same id, which is what you have defined here

Reply:
The `BCMinesActPermitCredential` and the `MinesActPermit` are 2 different things we are trying to define here. The permit number belongs to the `MinesActPermit`, which belongs to the `BCMinesActPermitCredential`

Reply:
As long as we have a definition of what is a `MinesActPermit`, we are good to go

Reply:
We will also have the terms of use included in the vc, binding its usage where this is clearly identified